addons/hostpath-storage/storage.yaml

```diff
@@ -17,6 +17,15 @@ spec:
         k8s-app: hostpath-provisioner
     spec:
       serviceAccountName: microk8s-hostpath
+      affinity:
```

Comment:
What is the purpose of this affinity rule? This "hostpath-test-node" does not look right.

Reply:
In multinode setup, we want the hostpath pod to be on the same node as the node from which the tests are executed, other wise `validate_storage()` and `validate_storage_custom_pvdir()` fail if the hostpath pod is created to some other node.

Reply:
The `storage.yaml` is what all users deploy. This affinity rule is only for our own tests. We should not be applying it to all deployments.

Reply:
Could we inject any affinity rules at the time of the test?

Reply:
Ok, lets do that.

Reply:
Done.